app/components/events/view/overview/event-tickets.js

```diff
@@ -1,5 +1,23 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 
 export default Component.extend({
-  classNames: ['ui', 'fluid', 'card']
+  classNames: ['ui', 'fluid', 'card'],
+  totalTickets : computed('data', function() {
+    return this.data.orderStat.tickets.completed + this.data.orderStat.tickets.placed;
+  }),
+  totalOrders : computed('data', function() {
+    return this.data.orderStat.orders.completed + this.data.orderStat.orders.placed;
+  }),
+  totalAmount : computed('data', function() {
+    return this.data.orderStat.sales.completed + this.data.orderStat.sales.placed;
+  }),
+  computedTicketData : computed('data.tickets', function() {
+    let tickets = this.data.tickets.toArray();
+    tickets.forEach(ticket => {
+      ticket.set('soldTickets', ticket.orderStatistics.tickets.completed + ticket.orderStatistics.tickets.placed);
```

Comment:
@niranjan94 this particular line is breaking the template in `event/overview`. I tried ticket.get('orderStatistics.tickets.completed`) since `orderStatistics` is a proxy but it's only returning `undefined`

Reply:
@niranjan94 I'm facing similar issue in one of my PR. How to handle a proxy amd retrieve value of an attribute?